pyubee/__init__.py

```diff
@@ -185,6 +185,33 @@ def authenticate(self, url, username, password):
         ),
         'authenticator': Evw3226Authenticator
     },
+    'DVW32CB': {
+        'url_session_active': '/main.asp',
+        'url_login': '/RgSwInfo.asp',
+        'url_logout': '/logout.asp',
+        # includes all devices, also WiFi
+        'url_connected_devices_lan': '/RgDhcp.asp',
+        'url_connected_devices_wifi': '/wlanAccess.asp',
+        'regex_login': re.compile(r'name="loginUsername"'),
+        'regex_wifi_devices': re.compile(
+            r'<tr bgcolor=#[0-9a-fA-F]+>'
+            r'<td>([0-9a-fA-F]{2}:[0-9a-fA-F]{2}:[0-9a-fA-F]{2}:[0-9a-fA-F]{2}:'  # mac address
+            r'[0-9a-fA-F]{2}:[0-9a-fA-F]{2})</td>'  # mac address, cont'd
+            r'<td>\d+</td>'  # age
+            r'<td>.+</td>'  # rssi
+            r'<td>.*</td>'  # ip address
+            r'<td>(.+)?</td>'  # hostname
```

Comment:
I would suggest to use `.*` instead of `.+` here.
I had issue with one phone connected to Ubee router, but not showing up in Home Assistant ([pull request #15](https://github.com/mzdrale/pyubee/pull/15/files)). It was because that phone didn't have hostname assigned in router's list of connected devices and regex didn't match.

Reply:
Done! How does it look now?

Reply:
I think you can remove the `?` now; i.e., `(.*)` is enough.

Reply:
Removed it! Thanks for your help